packages/editor/src/components/entities-saved-states/entity-type-list.js

```diff
@@ -16,20 +16,20 @@ import { store as coreStore } from '@wordpress/core-data';
  */
 import EntityRecordItem from './entity-record-item';
 
-function getEntityDescription( entity, length ) {
+function getEntityDescription( entity, count ) {
 	switch ( entity ) {
 		case 'site':
-			return _n(
-				'This change will affect your whole site.',
-				'These changes will affect your whole site.',
-				length
-			);
+			return 1 === count
+				? __( 'This change will affect your whole site.' )
+				: __( 'These changes will affect your whole site.' );
 		case 'wp_template':
-			return _n(
-				'This change will affect pages and posts that use this template.',
-				'These changes will affect pages and posts that use these templates.',
-				length
-			);
+			return 1 === count
+				? __(
+						'This change will affect pages and posts that use this template.'
+				  )
+				: __(
+						'These changes will affect pages and posts that use these templates.'
+				  );
```

Comment:
@ockham, considering that you can only modify one template at a time, I don't think we need this logic.

Reply:
Good point 👍

Reply:
Done [5805a49](https://github.com/WordPress/gutenberg/pull/40292/commits/5805a495bdd9e472218d8268e6ab1c71c83f3ee1) ✅